web/app/themes/mitlib-parent/css/scss/partials/_typography.scss

```diff
@@ -6,32 +6,95 @@ body {
 }
 
 h1 {
-	@include rem-first(font-size, 2.25);
+	@include rem-first(font-size, 1.75);
 	font-weight: 400;
+	line-height: 1.25;
 }
 
-h2 {
-	@include rem-first(font-size, 1.825);
+h2, h3, h4, h5, h6 {
 	font-weight: 400;
-	@include bp-tablet--landscape {
-		font-size: 2.125em;
-	}
-	a {
-		font-weight: 400;
-	}
+	margin: 15px 0 10px;
```

Comment:
This rule appears to impact the front page in a way we may not be comfortable with - there are h3s on the hours block which pick up this margin in a way that seems noteworthy. 

![Screenshot 2024-11-04 at 10 26 35 AM](https://github.com/user-attachments/assets/6f21d3a7-91eb-425c-866f-335c3bd31de2)

Reply:
Ahh yes, this must have changed in the latest iteration. I also noticed that the size of the H2s also shifted, so I'll probably need to untangle this a bit more.

From a practical standpoint, is it worth implementing the page-specific styles in the partial files of the appropriate name? For example, using `_home.scss` for the homepage specific heading styles? Keep the typography file completely global? I can move the page-specific overrides on like 46 of `_typography.scss`, too.

Either way, I think I'll need another round of iteration on this.

Reply:
Given that we have a `_home.scss` module already, I can see a good argument for putting any page-specific styles there, and leaving the basic thrust of your cleanup in place as you've built it so far. That would let us put all the rules for a given feature in the same place, rather than checking each base module for any overrides for a given feature.

Reply:
The other option, which I mention only for completeness and not because I favor it, would be to put the styles in the `mitlib-pull-hours` plugin which defines this specific widget. I'm not as big a fan of that, however, because that puts on the road to having styles in both the themes and in the various plugins which contribute to the UI - which feels worse from a maintenance perspective.

Reply:
Great, I'll lean into the modules for specific pages, I think that's the right approach.

I like the idea of trying to keep the distinction that plugins only provide content/functionality and lean on the theme for the styling (as much as we're able to). I suppose if we introduce something really novel and we're confident the styling should be consistent in all the places it appears, we could consider putting the styles with the plugin, but that seems like it could get messy quickly.